ProcessMaker/processmaker
# feature/FOUR-8705: Create Script Executor Details Empty

## Issue & Reproduction Steps
The ScriptExecutorUpdated does not have details

## Solution
avoid the event when there are no changes

## How to Test
1. Create a Script Executor
2. Edit the Script Executor
3. Go to Admin → Users
4. Check Security Logs

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-8705

## Code Review Checklist

ci:deploy
